src/app/components/site/Header.tsx

```diff
@@ -29,31 +27,23 @@ export const Header = () => {
               <div className="header-links ml-auto pr-3 px-md-3 d-flex align-items-center d-print-none">
                 {user &&
                   (!user.loggedIn ? (
-                    <React.Fragment>
-                      <div className="login mx-5 mx-sm-2">
-                        <Link to="/login">
-                          <span>Log in</span>
-                        </Link>
-                      </div>
-                      <div className="signup m-0 mr-md-4 ml-md-3">
-                        <Link to="/register">
-                          <span>Sign up</span>
-                        </Link>
-                      </div>
-                    </React.Fragment>
+                    <div className="login mx-4 mx-sm-2 d-flex flex-row">
```

Comment:
I'm trying to place why this still isn't quite right on mobile. It's because the font size should be 16px on mobile, not 18px, and that the search icon pink box is too big. It's currently 46x46 and should be 35x35. Are you ok to double check that please? :)

Reply:
I have corrected the sizing.
The sizing is no dynamic depending on the screen size.